src/commands/synthetics/crypto.ts

```diff
@@ -1,5 +1,4 @@
 import {generateKeyPairSync} from 'crypto'
-
```

Comment:
Our convention is to separate imports from Node API, then 3rd party libraries, then project imports.
Could we keep this blank line? (same for other removed blank lines in imports of other files of this PR)

Reply:
Ah missed that thanks, will do!